src/main/java/com/example/entityapi/EntityApiApplication.java

```diff
@@ -14,21 +13,29 @@
 
 @SpringBootApplication
 @EnableAutoConfiguration(exclude = ErrorMvcAutoConfiguration.class)
-public class EntityApiApplication {
-	public static void main(String[] args) {
-		SpringApplication.run(EntityApiApplication.class, args);
+public class EntityApiApplication
+{
+	public static void main(String[] args)
+	{
+		SpringApplication.run(EntityApiApplication.class,
+							  args
+							 );
```

Comment:
Your formatter is the Pizza Hut of formatters.

Reply:
eeesh thats pretty bad.